hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -9884,7 +9884,7 @@ objects:
       managed.openshift.io/gitHash: ${IMAGE_TAG}
       managed.openshift.io/gitRepoName: ${REPO_NAME}
       managed.openshift.io/osd: 'true'
-    name: osd-avo-resources-fedramp-vpce-acceptance
+    name: osd-avo-resources-fedramp-vpc-endpoints-us-gov-east-1
```

Comment:
Noted that this name change would probably leave a duplicate SSS behind that will need to be cleaned up